design-system/molecules/cards/card-project-marketplace/adapters/default/default.adapter.tsx

```diff
@@ -59,6 +65,71 @@ function Language({ id, name, percentage, nameClassNames = "" }: LanguageProps)
   );
 }
 
+function AvatarWithEcosystems({ name, logoUrl, ecosystems }: AvatarProps) {
+  const avatarRef = useRef<HTMLDivElement>(null);
+  const [avatarOffset, setAvatarOffset] = useState(0);
+
+  useEffect(() => {
+    if (avatarRef.current) {
+      setAvatarOffset(-avatarRef.current.offsetHeight / 2);
+    }
+  }, [avatarRef.current]);
+
```

Comment:
instead of settings a state you can memo it ?

Reply:
Tried, but the useMemo isn't reactive :/